### ISSUE TITLE
vagrant 1.9.1 pristine message from plug-in (sqlite3)

### ISSUE THREAD

**Issue by darkn3rd** [mitchellh/vagrant#8147]
I have some Vagrantfiles that use sqlite to enumerate systems and some configuration options.  The only way to make sqlite accessable was to run `vagrant plugin install sqlite3`.  Before this worked fine, but with Vagrant 1.9.1, I am  getting some message about using pristine and extensions not built.

### Vagrant version

`1.9.1`

### Host operating system

Mac OS X Sierra

### Guest operating system

n/a

### Vagrantfile

n/a

### Expected behavior

Should not get warning message.  This worked before without a problem.

### Actual behavior

When running a command like `vagrant status`, get this message:

```
Ignoring sqlite3-1.3.12 because its extensions are not built.  Try: gem pristine sqlite3 --version 1.3.12
```

### Steps to reproduce

```
vagrant plugin install sqlite3
vagrant status
```

**Comment by trappar** [comment]
I am also having this issue.

**Comment by chrispelzer** [comment]
I upgraded to vagrant 1.9.1 (also on macOS Sierra) and started experiencing this issue but with 2 different gems than the OP.

```
Ignoring hitimes-1.2.4 because its extensions are not built.  Try: gem pristine hitimes --version 1.2.4
Ignoring nio4r-2.0.0 because its extensions are not built.  Try: gem pristine nio4r --version 2.0.0
```

Running `gem pristine hitimes --version 1.2.4` will result with:
```
ERROR:  While executing gem ... (Gem::Exception)
    Failed to find gems ["hitimes"] = 1.2.4
```

Installing either gem to the version it asks for doesn't work either.

Looking at the gems within the folder `/opt/vagrant/embedded/gems/gems` shows the gems not there.

I haven't noticed any negative effects except the messages appearing whenever running `vagrant`

**Comment by trappar** [comment]
The gems are located inside the `~/.vagrant.d` directory (can't remember the exact location off the top of my head). I've tried everything I can think of to run commands against that location but nothing works.

**Comment by peterdragon** [comment]
+1
$ vagrant version
Ignoring hitimes-1.2.4 because its extensions are not built.  Try: gem pristine hitimes --version 1.2.4
Ignoring nio4r-2.0.0 because its extensions are not built.  Try: gem pristine nio4r --version 2.0.0
Installed Version: 1.9.1
Latest Version: 1.9.1

$ rbenv versions
  system
  2.2.3
* 2.3.1 (set by /Users/peter/.rbenv/version)

$ ls ~/.vagrant.d/gems/2.2.5/extensions/x86_64-darwin-13/2.2.0/
hitimes-1.2.4 nio4r-2.0.0

**Comment by chrispelzer** [comment]
@peterdragon Do you use the plugin vagrant-dns? 

Those 2 gems for me only show up if I have vagrant-dns installed. The issue is on the plugin not being up to date (last commit was 3 years ago) to work with the bundler changes on vagrant 1.9.0+

There is an issue on the repo, but not sure if the package is maintained anymore. https://github.com/BerlinVagrant/vagrant-dns/issues/46

I might have to switch over to landrush when they come out with their patch.
https://github.com/vagrant-landrush/landrush/issues/295

**Comment by chrisroberts** [comment]
Fixed via #8191
